packages/artillery/test/cli/command-run.test.js

```diff
@@ -288,6 +285,47 @@ tap.test(
   }
 );
 
+tap.test(
+  'Ramp to script throughput behaves as expected running on multiple workers 1s duration',
+  async (t) => {
+    // amount of workers was still affecting ramps with duration = 1s
+    // check single worker and multiple workers now generate same throughput
+    const totalRequests = 10;
+
+    const reportMultipleFile = 'multiple_workers.json';
+    const reportMultipleFilePath = await getRootPath(reportMultipleFile);
+
+    const reportSingleFile = 'single_worker.json';
+    const reportSingleFilePath = await getRootPath(reportSingleFile);
+
+    const [exitCodeMultiple] = await execute([
+      'run',
+      '-o',
+      reportMultipleFile,
+      'test/scripts/ramp-regression-1682.json'
+    ]);
```

Comment:
Should this set `WORKERS` explicitly to avoid false positives on machines with only one CPU?